components/breadcrumb/_breadcrumb.scss

```diff
@@ -2,20 +2,25 @@
   font-size: 0.7rem;
   display: flex;
   align-items: center;
-  padding: 1rem 1rem;
+  padding: 1rem 0;
 }
 .breadcrumb__items {
-
+  margin: 0;
+  padding: 0;
 }
 .breadcrumb__item {
   display: inline-block;
+  &:after {
+    content: ">"; /* Divider between breadcrumb items */
```

Comment:
Are we happy with how this will render across browsers and operating systems? Option would be to use an svg and png fallback.

Reply:
@boxadesign good point. We’ll pick that up tomorrow.
Also noticed that there is an odd font size specified on the wrapper.